specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/examples/AssessedMachines_Get.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "resourceGroupName": "abgoyal-westEurope",
+    "api-version": "2019-05-01",
+    "subscriptionId": "6393a73f-8d55-47ef-b6dd-179b3e0c7910",
+    "projectName": "abgoyalWEselfhostb72bproject",
+    "groupName": "Test1",
+    "assessmentName": "assessment_5_9_2019_16_22_14",
+    "assessedMachineName": "f57fe432-3bd2-486a-a83a-6f4d99f1a952"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/6393a73f-8d55-47ef-b6dd-179b3e0c7910/resourceGroups/abgoyal-westeurope/providers/Microsoft.Migrate/assessmentprojects/abgoyalWEselfhostb72bproject/groups/Test1/assessments/assessment_5_9_2019_16_22_14/assessedMachines/f57fe432-3bd2-486a-a83a-6f4d99f1a952",
+        "name": "f57fe432-3bd2-486a-a83a-6f4d99f1a952",
+        "eTag": "\"b300e5dd-0000-0d00-0000-5cd4065f0000\"",
+        "type": "Microsoft.Migrate/assessmentprojects/groups/assessments/assessedMachines",
+        "properties": {
+          "createdTimestamp": "2019-05-09T10:52:15.7789159Z",
+          "updatedTimestamp": "2019-05-09T10:52:15.7789159Z",
+          "bootType": "BIOS",
+          "displayName": "SHubhamVMNew",
+          "description": "Microsoft Azure Migration Image on Windows Server 2016",
+          "datacenterMachineArmId": "/subscriptions/6393a73f-8d55-47ef-b6dd-179b3e0c7910/resourcegroups/abgoyal-westeurope/providers/microsoft.offazure/vmwaresites/portalvcenterbc2fsite/machines/idclab-a360-fareast-corp-micros-86617dcf-effe-59ad-8c3a-cdd3ea7300d3_52e1f68c-bea5-19ff-d0ad-6a94b79a286f",
+          "datacenterManagementServerArmId": "/subscriptions/6393a73f-8d55-47ef-b6dd-179b3e0c7910/resourceGroups/abgoyal-westEurope/providers/Microsoft.OffAzure/VMwareSites/PortalvCenterbc2fsite/vcenters/idclab-a360-fareast-corp-micros-86617dcf-effe-59ad-8c3a-cdd3ea7300d3",
+          "datacenterManagementServerName": "IDCLAB-A360.fareast.corp.microsoft.com",
+          "megabytesOfMemory": 16384.0,
+          "numberOfCores": 8,
+          "operatingSystemType": "windowsGuest",
+          "operatingSystemName": "Microsoft Windows Server 2016 (64-bit)",
+          "operatingSystemVersion": null,
+          "monthlyBandwidthCost": 0.0,
+          "monthlyStorageCost": 5.888,
+          "montlyPremiumStorageCost": 0.0,
+          "monthlyStandardSSDStorageCost": 0.0,
+          "confidenceRatingInPercentage": 0.0,
+          "disks": {
+            "6000C299-02f5-d137-9bab-8a8ee7b192a0": {
```

Comment:
>6000C299-02f5-d137-9bab-8a8ee7b192a0 [](start = 13, length = 36)

GUIDs typically have horrible usability. Is there another identifier we could use for disks? Does it map to something that you can find on the machine?

Reply:
This is actually a UUID (Universally unique identifier) that we are getting from another service that is the discovery service. The discovery service gets it from VMware APIs so we are not adding anything from our end here. Moreover, this was the field that we found to be unique amongst all the properties, therefore, choosing this.

Reply:
Ok.

---
In reply to: [325075829](https://github.com/Azure/azure-rest-api-specs/pull/7063#discussion_r325075829) [](ancestors = 325075829)